site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-siteInfo-list.vm

```diff
@@ -771,7 +772,13 @@ function printPreview(target){
 			</div>
 			<input type="hidden" name="sakai_csrf_token" value="$sakai_csrf_token" />
 		</form>
-		
+		#if($siteDuplicated)
+		<div id="newSiteAlert" class="$dupeSiteStatusID">
+		<span class="newSiteAlertMessage">$tlang.getString("sitdup.dupsit")<a title="$validator.escapeHtml($duplicatedName)" href="/portal/site/$dupeSiteStatusID" target="_top"> $validator.escapeHtml($duplicatedName)</a> $tlang.getString("sitdup.hasbeedup") </span>
```

Comment:
You shouldn't hardcode paths like. You should use the method `Site.getUrl()` as it can be that the URL to the site is somewhere else.